gui/webdisplay/src/TWebWindowsManager.cxx

```diff
@@ -340,7 +344,7 @@ void ROOT::Experimental::TWebWindowsManager::Unregister(ROOT::Experimental::TWeb
 //////////////////////////////////////////////////////////////////////////
 /// Provide URL address to access specified window from inside or from remote
 
-std::string ROOT::Experimental::TWebWindowsManager::GetUrl(ROOT::Experimental::TWebWindow &win, bool remote)
+std::string ROOT::Experimental::TWebWindowsManager::GetUrl(ROOT::Experimental::TWebWindow &win, bool batch_mode, bool remote)
```

Comment:
Consider adding new functions for the boolean flags, or enums. `GetUrl(win, true, false)` is *really* hard to read.

Reply:
Done